
try:
    print(1+'1')
except Exception as e:
    print("error happen")

print("Hello")
